### PR TITLE
Dashboard us7

### PR DESCRIPTION
**Pull request by MiketheBite**

### CODE REVIEW

File: components/Dashboard/ProfileDashboard.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <ProfileContainer>
+      <ProfileImage
+        width={76.8}
+        height={76.8}
+        src={nurseLeader}
+        alt="Nurse Photo"
+      />
+      <div>
+        <Welcome>Welcome</Welcome>
+        <ProfileName>Thomas Bauer</ProfileName>
```
Comment (Kheirah) [inline components/Dashboard/ProfileDashboard.js:21]: Is this supposed to stay "Thomas Bauer"?

File: components/Dashboard/ShiftsDetails.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        const response = await fetch("/api/nurses");
+        const data = await response.json();
+        setNursesList(data);
+      } catch (error) {
+        console.error("Error fetching nurses:", error);
+      }
+    }
+
+    fetchNurses();
+  }, []);
```
Comment (Kheirah) [inline components/Dashboard/ShiftsDetails.js:32]: Is there a reason why you're not using the SWR hook here? When using fetch, you potentially make unnecessary requests. When using SWR, the hook makes sure not to overfetch even when it's called multiple times.

**Review by Kheirah — APPROVED** [review]
Very well done!

Just a few comments to look into, see below.

File: components/Dashboard/ProfileDashboard.js (hunk 1 above)
Comment (MiketheBite) [reply components/Dashboard/ProfileDashboard.js:21]: It's supposed to be the name given after login/authentication.

**Pull request merged**